src/tutorial/counter-app/multiple-functions.md

```diff
@@ -30,7 +30,7 @@ By defining custom types and aliases, we can simplify our code and make it more
 ```rust
 type Err = Box<dyn std::error::Error>;
 type Result<T> = std::result::Result<T, Err>;
-pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+pub type Frame<'a> = ratatui::Frame<'a>;
```

Comment:
I believe we can drop this type definition entirely (notice the LHS and RHS are the same).

We'll have to update the imports in an earlier block to bring `Frame` into scope, i.e. `use ratatui::prelude::Frame`.